lib/cubit/payments/payments_state.dart

```diff
@@ -16,8 +17,18 @@ class PaymentsState {
     List<PaymentData>? payments,
     PaymentFilters? paymentFilters,
   }) {
+    // This is a workaround to only include BTC assets in the unfiltered payments.
+    // If Misty is to support multi-assets then this prefilter should be removed.
+    final List<PaymentData>? prefilteredPayments = payments?.where((PaymentData paymentData) {
+      final String? paymentAssetTicker = paymentData.details.map(
+        liquid: (PaymentDetails_Liquid details) => details.assetInfo?.ticker ?? '',
+        orElse: () => null,
+      );
+      return paymentAssetTicker == null || paymentAssetTicker == 'BTC';
```

Comment:
Given there might be multiple assets with this ticker, isn't it dangerous to use it here? A user could think he's getting L-BTC but he would be getting something else.

Reply:
We're only using the default configured asset metadata, so the only available tickers are 'BTC' and 'USDt'

Reply:
Ah, right! Thanks